.github/env.yml

```diff
@@ -0,0 +1,9 @@
+TEAM_NAME: Teacher Services Infra
```

Comment:
This is specific to workflows, so it may make sense to move it under .github/workflows ?

Reply:
I wasn't comfortable with that, as the github system seems to consider yaml files in workflows as possible workflow files. so I took my lead from labeller.yml which has the workflow file where you would expect it, but the controlling yaml file in .github

Reply:
yes, github could try and treat yml files inside workflows/ as a workflow definition file